### PR TITLE
fix(specs): update required for kotlin tests to pass

### PR DESCRIPTION
**Pull request by aallam**
## 🧭 What and Why

Fixes necessary API/CTS specifications for Kotlin API client to pass

### Changes included:

- CTS specs
- API specs

### CODE REVIEW

File: tests/CTS/methods/requests/ingestion/createTask.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
       "sourceID": "search",
       "destinationID": "destinationName",
       "trigger": {
-        "type": "subscription"
```
Comment (shortcuts) [inline tests/CTS/methods/requests/ingestion/createTask.json:56]: Both exist, it doesn't work?

Comment (aallam) [reply tests/CTS/methods/requests/ingestion/createTask.json:56]: Nope :/ The CTS generator consistently [chooses](https://github.com/algolia/api-clients-automation/blob/7032ca41177d911c8dacba664a9bd79e22661bfc/generators/src/main/java/com/algolia/codegen/cts/tests/ParametersWithDataType.java#L269) the wrong type! Ideally, we should fix the CTS generator to properly handle this case, but that might be time-consuming. So, it's a problem to tackle another day!

Comment (shortcuts) [reply tests/CTS/methods/requests/ingestion/createTask.json:56]: Make sense ty for the explanation!

**Review by shortcuts — APPROVED** [review]
Gg!!

**Pull request merged**